Atlassian Bitbucket Server integration for Jenkins blog post and use case (#2703)
jenkins-infra/jenkins.io: content/blog/2019/12/2019-12-13-atlassians-new-bitbucket-server-integration-for-jenkins.adoc

---
layout: post
title: "Atlassian's new Bitbucket Server integration for Jenkins"
tags:
- bitbucket
- pipeline
- plugins
author: dkjellin
---

> We know that for many of our customers Jenkins is incredibly important and its integration with link:https://www.atlassian.com/software/bitbucket/enterprise/data-center[Bitbucket Server] is a key part of their development workflow. Unfortunately, we also know that integrating Bitbucket Server with Jenkins wasn’t always easy – it may have required multiple plugins and considerable time. That’s why earlier this year we set out to change this. We began building our own integration, and we’re proud to announce that v1.0 is out.

Author: Good tip. I'll do this moving forward. Thanks.

Our new integration is the easiest way to link Jenkins with Bitbucket Server. It streamlines the entire set-up process, from creating a webhook to trigger builds in Jenkins, to posting build statuses back to Bitbucket Server. It also supports link:https://confluence.atlassian.com/bitbucketserver/smart-mirroring-776640046.html[smart mirroring] and lets Jenkins clone from mirrors to free up valuable resources on your primary server.

This link:https://plugins.jenkins.io/atlassian-bitbucket-server-integration[Atlassian-supported integration] is available to install through Jenkins now. Watch this video to find out how, or read our link:https://hello.atlassian.net/wiki/spaces/BSERV/pages/459549289/Bitbucket+Server+exploration+sparring[use-case article] to learn more about it.

> video::0-FugzVYJQU[youtube, align="center"]

Author: That link was my mistake. Sorry about that. I've updated it now. 

Which references do you think we should add here?

> video::0-FugzVYJQU[youtube, align="center"]

Author: Hi @oleg-nenashev . I haven't heard from you so I thought I'd check. Which references do you think we should add here?

---
layout: post
title: "Atlassian's new Bitbucket Server integration for Jenkins"
tags:
- bitbucket
- pipeline
- plugins
author: dkjellin
---

We know that for many of our customers Jenkins is incredibly important and its integration with link:https://www.atlassian.com/software/bitbucket/enterprise/data-center[Bitbucket Server] is a key part of their development workflow. 
Unfortunately, we also know that integrating Bitbucket Server with Jenkins wasn’t always easy – it may have required multiple plugins and considerable time. 
That’s why earlier this year we set out to change this. 
We began building our own integration, and we’re proud to announce that v1.0 is out.

> The new link:https://plugins.jenkins.io/atlassian-bitbucket-server-integration[Bitbucket Server integration for Jenkins plugin], which is built and supported by Atlassian, is the easiest way to link Jenkins with Bitbucket Server. 

Reviewer: ```suggestion
The new plugin:atlassian-bitbucket-server-integration[Bitbucket Server integration for Jenkins plugin], which is built and supported by Atlassian, is the easiest way to link Jenkins with Bitbucket Server. 
```

It streamlines the entire set-up process, from creating a webhook to trigger builds in Jenkins, to posting build statuses back to Bitbucket Server. 
It also supports link:https://confluence.atlassian.com/bitbucketserver/smart-mirroring-776640046.html[smart mirroring] and lets Jenkins clone from mirrors to free up valuable resources on your primary server.

Our plugin is available to install through Jenkins now. 

> Watch this video to find out how, or read our link:https://jenkins.io/solutions/bitbucketserver/[use-case article] to learn more about it.

Reviewer: ```suggestion
Watch this video to find out how, or read the link:https://jenkins.io/solutions/bitbucketserver/[BitBucket Server solution page] to learn more about it.
```
